platform/ios/INSTALL.md

```diff
@@ -1,6 +1,8 @@
-# Integrating custom builds of the Mapbox iOS SDK into your application
+# Integrating the Mapbox iOS SDK into your application
```

Comment:
In #7806, I [went in the opposite direction](https://github.com/mapbox/mapbox-gl-native/pull/7806/files#diff-2aea5ea0fef9040989e9f539f9ab2fb4L1) for the macOS SDK. Let’s synchronize the two documents. Now that both SDKs have official homepages, both INSTALL.mds serve the same purpose.